30-indoor-air-quality-and-gas-leak/firmware/sparrow-application/aqi/sfe_ens160.cpp

```diff
@@ -0,0 +1,800 @@
+#include "sfe_ens160.h"
+
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
@m-mcgowan is this your code? If so, I'll review.

Reply:
No, it's from the Sparkfun ENS160 Arduino library. They had abstracted the comms bus, so it was fairly easy to integrate.